deploy/templates/nstemplatetiers/appstudio/ns_appstudio.yaml

```diff
@@ -12,6 +12,7 @@ objects:
       openshift.io/requester: ${USERNAME}
     labels:
       name: ${USERNAME}
+      toolchain.dev.openshift.com/workspace: ${USERNAME}
```

Comment:
Is it possible to add a check in the unit test that this label is equal to the one that is defined in our API repo? https://github.com/codeready-toolchain/api/blob/master/api/v1alpha1/space_types.go#L12
You know, just in case that someone from another AppStudio team wanted to use the constant in their code...

Reply:
It's checked in the e2e tests: https://github.com/codeready-toolchain/toolchain-e2e/pull/433/files#diff-b93768d40b5d9e46aafbc8d45fe064e80bf7edb13907f41a0c2dd470b850b636R63-R64
We did agree on keeping the unit tests for the tier content as simple/small as possible (basically just checking that the tiers are present) and do all the content checks in the e2e tests, right?

Reply:
OK, I didn't take a look at the e2e tests. 
Then I'm fine with it :+1:

Reply:
yes, I thought it would be simpler/more straightforward to just add a check in the e2e tests 🤷‍♂️